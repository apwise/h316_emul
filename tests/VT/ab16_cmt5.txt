ptr ../../tapes/VT/AB16-CMT5_slst.ptp
g'1
cl
limit 200000000
g'1000
q
