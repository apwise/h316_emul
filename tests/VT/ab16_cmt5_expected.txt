
MON> ptr ../../tapes/VT/AB16-CMT5_slst.ptp
MON> g'1

MON> cl
MON> limit 200000000
MON> g'1000



AB16-CMT5   JUN 7, 73   REV E 
CP IS 316 



PS0001  001673  077777  
PS0002  000021  076117  
0205062896: limit reached

MON> q
